packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -82,25 +82,25 @@ function Example() {
 ### Size
```

Comment:
Still need to document using `<Item><Heading>blah</Heading></Item>` to make Breadcrumbs multiline but waiting for feedback if the multiline behavior should also apply if the user passes `isHeading=true` without including `<Heading>`